youtube_dl/extractor/common.py

```diff
@@ -2194,7 +2194,11 @@ def geo_verification_headers(self):
         return headers
 
     def _generic_id(self, url):
-        return compat_urllib_parse_unquote(os.path.splitext(url.rstrip('/').split('/')[-1])[0])
+        id = compat_urllib_parse_unquote(os.path.splitext(url.rstrip('/').split('/')[-1])[0])
+        # If our id matches a stopword, use the 2nd-to-last URL path component instead
+        if id in ['story']:
+            id = compat_urllib_parse_unquote(os.path.splitext(url.rstrip('/').split('/')[-2])[0])
```

Comment:
Well, `story` was the only example I could find. I assume others would add more words to this list as they came up.
I hadn't appreciated that the id did not appear in the output filename, I see your point. (It does appear in the debug logs, e.g. --write-pages and soforth, but not in the final video output). That does make this seem not worth the effort.